pkg/network/kuryr.go

```diff
@@ -113,7 +113,13 @@ func renderKuryr(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.BootstrapR
 	data.Data["WebhookKey"] = b.WebhookKey
 
 	// Nodes Network MTU
-	data.Data["NodesNetworkMTU"] = b.NodesNetworkMTU
+	mtu  := uint32(b.NodesNetworkMTU)
+	if c.MTU != nil && *c.MTU != 0 {
+		if *c.MTU < mtu {
```

Comment:
I think I'd rather report an error in that situation instead of trying to fix user's mistake.